prime-router/docs/design/design/yaml-validation/yaml-cli-task.md

```diff
@@ -0,0 +1,11 @@
+# YAML CLI Task
+
+## Approach
+
+We will create a CLI task that will validate all our YAML files at one time. This will be crucial as a part of
```

Comment:
Update to answer these Qs:
- What kind of validation? You have two documents called structure-validation and valie-validation, do you mean both? Maybe hyperlink to those in here when you mention them?
- What do you mean by all our YAML files. HL7->FHIR mappings, FHIR Transforms, and org settings?

Reply:
To answer your questions:

What kind of validation? You have two documents called structure-validation and value-validation, do you mean both? Maybe hyperlink to those in here when you mention them?

As a part of the CLI task I want to run both structure and value validations. I'll add a hyperlink in the doc.

What do you mean by all our YAML files. HL7->FHIR mappings, FHIR Transforms, and org settings?\

Yes those are the files I mean. I can list them out in the doc.